.github/workflows/auto-merge.yaml

```diff
@@ -3,11 +3,21 @@ name: Auto-merge approved PRs
 on:
   pull_request_review:
     types: [submitted]
+  pull_request_target:
+    types: [labeled, synchronize]
+  workflow_dispatch:
```

Comment:
Delete the workflow dispatch trigger here. This should allow us to remove the extra steps introduced below